packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -4188,6 +4219,15 @@ class SemanticsConfiguration {
     }
   }
 
+  /// An identifier of the owning [RenderObject] used for UI testing. This value
```

Comment:
This doc seems a bit confusing. The UIAutomator interact with semantics node, not renderObject.

Reply:
Good catch, thank you. I'm still new to this code.

Reply:
Here I took inspiration from other `SemanticsConfiguration` fields, e.g.:

https://github.com/flutter/flutter/blob/0b98178af46c82d48dd6e008290e23f8a21f122a/packages/flutter/lib/src/semantics/semantics.dart#L4231-L4240

What'd you change?